csvs/bitget-spot-jst.json

```
@@ -158,7 +205,101 @@
         ]
       },
       {
-        "description": "Ignore (Buy, Sell, Financial, Transfer in, Transfer out, 365, Interest)",
+        "description": "LaunchX income",
+        "conditions": [
+          {
+            "value": { "type": "cell", "column": "type" },
+            "pattern": "LaunchX income"
+          }
+        ],
+        "generators": [
+          {
+            "type": "gift",
+            "source": {
+              "type": "join",
+              "values": [
+                { "type": "service" },
+                "_spot_",
+                { "type": "cell", "column": "order" }
+              ]
+            },
+            "timestamp": { "type": "cell", "column": "date" },
+            "comment": {
+              "type": "join",
+              "values": [
+                "type: ",
+                { "type": "cell", "column": "type" },
+                " / coin: ",
+                { "type": "cell", "column": "coin" },
+                " / amount: ",
+                { "type": "cell", "column": "amount" }
+              ]
+            },
+            "transfers": [
+              {
+                "direction": "in",
+                "amount": {
+                  "type": "text",
+                  "value": { "type": "cell", "column": "amount" }
+                },
+                "asset": {
+                  "type": "symbol",
+                  "value": { "type": "cell", "column": "coin" }
+                }
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "description": "LaunchX expense",
+        "conditions": [
+          {
+            "value": { "type": "cell", "column": "type" },
+            "pattern": "LaunchX expense"
+          }
+        ],
+        "generators": [
+          {
+            "type": "transfer",
```

```suggestion
            "type": "fee",
```

LaunchX は実質トークンの購入（IEOに近い）取引っぽいので、購入に使用したトークンは全額経費計上できる処理にした方が良いかも

（本当は LaunchX 受け取り側の通貨との売買取引にするのが最適だと思うけれど）

---

https://github.com/0xtorch/datasource/pull/9245/commits/f5500a14b8706f0e1b7706d341c7ef0004806883
で似た取引も追加して fee に変更